larray/tests/test_array.py

```diff
@@ -254,7 +254,7 @@ def test_getitem(self):
         assert_array_equal(la[..., lipro159], raw[..., [0, 4, 8]])
 
         # string 'int..int'
-        assert_array_equal(la['10..15'], la['10,11,12,13,14,15'])
+        assert_array_equal(la['8, 10..13, 15'], la['8,10,11,12,13,15'])
```

Comment:
I would rather have a test for both (with and without space), we never know how we will break things in the future :)